android-components/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineTest.kt

```diff
@@ -918,7 +919,7 @@ class GeckoEngineTest {
         var onErrorCalled = false
         val result = GeckoResult<GeckoWebExtension>()
 
-        whenever(extensionController.install(any())).thenReturn(result)
+        whenever(extensionController.install(any(), any())).thenReturn(result)
```

Comment:
we probably want to capture a `null` somewhere in this file (in an existing test) for when the install method is not specified.